requirements.txt

```diff
@@ -11,3 +11,4 @@ bblfsh==2.12.7
 requirements-parser==0.2.0
 lookout-sdk==0.4.1
 packaging==19.0
+numpy==1.16.1
```

Comment:
numpy is already required in Modelforge - and actually contradicts with 1.16.1. Do we need any feature from 1.16?

Reply:
no, I need whatever numpy. let's remove it.